content/administration/maintain/odoo_online.rst

```diff
@@ -115,8 +117,8 @@ Delete a database instantly.
    Deleting a database means that all data is permanently lost. The deletion is instant and for all
    users. It is recommended to create a backup of the database before deleting it.
 
-Read carefully the warning message that pops up and proceed only if you fully understand the
-implications of deleting a database:
+Carefully read the warning message that pops up and proceed only if the implications of deleting a
+database are fully understood:
```

Comment:
Reminder to break the dependency between written instruction and images; the text needs to stand on it's own without relying on images to convey important information.

Can take care of this on another PR if you want since it's outside the scope of this PR.